src/components/pages/SearchResultsPage.tsx

```diff
@@ -222,21 +232,45 @@ const SearchResultsPage = ({ location }: { location: Location }) => {
                   </Container>
                 </Responsive>
               </Grid.Row>
+              {!showPlaceholder &&
+                showPagination &&
+                searchResults &&
+                searchResults.length > 0 && (
+                  <ResponsiveRow>
+                    <Grid.Column
+                      computer={12}
+                      tablet={12}
+                      mobile={16}
+                      largeScreen={12}
+                      widescreen={12}
+                      style={{ display: 'flex', justifyContent: 'center' }}
+                    >
+                      <Grid style={{ width: '100%' }}>
+                        <Grid.Row centered>
+                          <Grid.Column
+                            mobile="12"
+                            tablet="6"
+                            computer="6"
+                            largeScreen="4"
+                            widescreen="4"
```

Comment:
I'm not sure which one is the correct one, but since the others is using `{}`, probably it'll become like below?
```suggestion
                            mobile={12}
                            tablet={6}
                            computer={6}
                            largeScreen={4}
                            widescreen={4}
```

Reply:
Oh, my bad, both are okay but I will edit it with the {} notation